components/cards/src/css/cards.css

```diff
@@ -0,0 +1,345 @@
+@import url("../../../../css/components.css");
+
+body {
+	font-family: inherit;
+}
+.flex-jc-ac {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-wrap: wrap;
+}
```

Comment:
try to break this into three parts so that user can especially use any one property